### PR TITLE
feat(@angular-ru/common): add Includes pipe

### PR DESCRIPTION
**Pull request by Schirbak**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [x] The commit message follows our guidelines: https://github.com/ngxs/store/blob/master/CONTRIBUTING.md#commit
-   [x] Tests for the changes have been added (for bug fixes / features)
-   [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by nikitasadchikov — APPROVED** [review]

**Review by mansurishmuhametov — APPROVED** [review]

File: packages/common/pipes/src/includes/includes.pipe.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({ name: 'includes' })
+export class IncludesPipe implements PipeTransform {
+    public transform<T>(array: T[] | null | undefined, entry: T): boolean {
+        return array?.includes(entry) ?? false;
```
Comment (splincode) [inline packages/common/pipes/src/includes/includes.pipe.ts:6]: Maybe we can create util `includes` from `@angular-ru/common/array`

```ts
public transform<T>(array: T[] | null | undefined, entry: IncludeElement<T>): boolean {
        return includes(array, entry);
}
```

```ts
export type IncludeEntry<T> =  T | T[] | RegExp;

export function includes<T>(T[] | null | undefined, entry: IncludeEntry<T>): boolean {
  // what do you think to improve the ability to search across multiple parameters?
}
```

Comment (Schirbak) [reply packages/common/pipes/src/includes/includes.pipe.ts:6]: i think it's all — a different functions.
`includesAny`, `includesAll`, `includesMatching`

Comment (splincode) [reply packages/common/pipes/src/includes/includes.pipe.ts:6]: good point, do you want implements it’s?

Comment (Schirbak) [reply packages/common/pipes/src/includes/includes.pipe.ts:6]: not here. moreover i think to use this functions as a pipe will complicate a template and you should find another way to calculate it. in short i'm against to make pipes from this

Comment (splincode) [reply packages/common/pipes/src/includes/includes.pipe.ts:6]: sure, create issue about your proposal

**Review by splincode — APPROVED** [review]

**Pull request merged**